### PR TITLE
Add login page and welcome page CSS

### PR DESCRIPTION
**Pull request by SNsir**

### CODE REVIEW

**Review by h-haddad — CHANGES_REQUESTED** [review]

File: galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/VLoginBox.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    font-style: normal;
+    font-size: 1.2em;
+    min-width: 152px;
+    margin: 0;
+    border: #dadada 1px solid;
+}
+
+* {
+  box-sizing: border-box;
+}
```
[the remark this answers is not in the document]
Comment (SNsir) [reply galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/VLoginBox.css:144]: I couldn't add style when focused

File: galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/VHeader.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+#header #welcome {
+    margin-left: auto;
+    margin-right: var(--lumo-space-m);
+}
+
+#moduleList {
+    display: block;
+    max-width: 85%;
+    margin-left: 0;
+    height: 63px;
```
[the remark this answers is not in the document]
Comment (SNsir) [reply galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/VHeader.css:36]: we need margin-left

File: galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/VHeader.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    --lumo-font-family: Arial, sans-serif;
+    --lumo-primary-text-color: var(--lumo-shade-70pct);
+    --lumo-primary-color: var(--lumo-shade-50pct);
+    --lumo-font-size-m: 0.800rem;
+    border-right: 1px solid #dadada;
+    height: 63px;
+    box-sizing: border-box;
+}
+
+#moduleList :not(#user_menu):first-child::part(menu-bar-button):hover {
```
[the remark this answers is not in the document]
Comment (SNsir) [reply galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/VHeader.css:80]: we will not use the >> icon , and the color is already chosen (gray)

File: galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/VHeader.css [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+/*
+ * Right Menus style
+ */
+#user_menu , #wrkcontext_menu , #bookmarks_menu , #admin_menu {
+    background-color: var(--lumo-tint);
+    margin: 0;
+    box-sizing: border-box;
+    padding: var(--lumo-space-xxs);
+}
```
[the remark this answers is not in the document]
Comment (SNsir) [reply galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/VHeader.css:113]: yes we need it

File: galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/Form.css [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    vertical-align: bottom;
+    font-family: Arial, sans-serif;
+}
+
+/*
+ *  Anchor's style
+ */
+#session span a {
+    text-decoration: none;
+}
```
[the remark this answers is not in the document]
Comment (SNsir) [reply galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/Form.css:45]: yes we need it to delete underline

File: galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/Form.css [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+    vertical-align: bottom;
+    font-size: 16px;
+    line-height: normal;
+    --lumo-space-m: 0;
+}
+
+html {
+    --lumo-space-m: 0;
+}
+
```
[the remark this answers is not in the document]
Comment (SNsir) [reply galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/Form.css:129]: yes it is !

File: galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/Report.css [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+/*
+ * Report style
+ */
+::part(cell header-cell) {
+    --lumo-font-family:  Arial, sans-serif;
+    --lumo-primary-text-color: var(--lumo-error-color);
+     background: #f2f2f2;
+}
```
[the remark this answers is not in the document]
Comment (SNsir) [reply galite-core/src/main/resources/META-INF/resources/frontend/styles/galite/Report.css:24]: we need the background color

**Review by h-haddad — CHANGES_REQUESTED** [review]

File: galite-core/src/main/kotlin/org/kopi/galite/ui/vaadin/main/MainWindow.kt [hunk 8]
```diff
@@ -77,7 +80,14 @@ class MainWindow(locale: Locale, val logo: String, val href: String) : AppLayout
 
     content.setContent(container)
     addToNavbar(header)
-    header.setWelcome(welcome)
+    val welcomeContainer = Div()
+    welcomeContainer.setId("welcomeContainer")
```
Comment (hfazai) [inline galite-core/src/main/kotlin/org/kopi/galite/ui/vaadin/main/MainWindow.kt:84]: `welcome-container`

File: galite-core/src/main/kotlin/org/kopi/galite/ui/vaadin/main/MainWindow.kt [hunk 9]
```diff
@@ -77,7 +80,14 @@ class MainWindow(locale: Locale, val logo: String, val href: String) : AppLayout
...
+    val horizontalAlignContainer = Div()
+    horizontalAlignContainer.setId("horizontalAlignContainer")
```
Comment (hfazai) [inline galite-core/src/main/kotlin/org/kopi/galite/ui/vaadin/main/MainWindow.kt:86]: `horizontal-align-container`

**Review by h-haddad — APPROVED** [review]

**Pull request merged**